### PR TITLE
Add on_delete option to belongs_to

### PR DESCRIPTION
**Pull request by mikeeus**
Related to #31.

This duplicates the `ALLOWED_ON_DELETE_STRATEGIES` const from the CreateForeignKeyStatement class. I'm not sure how to refactor this, maybe if there was a helper class that holds these constants. What do you think?

### CODE REVIEW

File: src/lucky_migrator/create_table_statement.cr [hunk 1]
```diff
@@ -148,11 +152,15 @@ class LuckyMigrator::CreateTableStatement
...
   end
 
-  def references(table_name : String | Symbol | Nil)
-    if table_name
+  def references(table_name : String | Symbol | Nil, on_delete = :do_nothing)
+    if table_name.nil?
+      ""
+    elsif on_delete == :do_nothing
       " REFERENCES #{table_name}"
+    elsif ALLOWED_ON_DELETE_STRATEGIES.includes?(on_delete)
```
Comment (paulcsmith) [inline src/lucky_migrator/create_table_statement.cr]: You can get at the constant in another class like this `CreateForeignKeyStatement::ALLOWED_ON_DELETE_STRATEGIES`

Comment (mikeeus) [reply src/lucky_migrator/create_table_statement.cr]: ah of course! Thanks

**Comment by mikeeus** [comment]
Hows this one?

**Comment by mikeeus** [comment]
K I added it to a migration

**Comment by paulcsmith** [comment]
🎉

**Pull request merged**